src/tasks/run.js

```diff
@@ -135,7 +135,7 @@ module.exports = function() {
   const webpackConfig = require('./webpack.viewSandbox.config');
 
   const webpackMiddlewareOptions = {
-    stats: 'errors-only'
+    stats: 'minimal'
```

Comment:
Changed this to minimal because the latest Webpack shows something like this: 
```
ℹ ｢wdm｣:    59 modules
ℹ ｢wdm｣: Compiled successfully.
```

With `errors-only` it would show something like:

```
ℹ ｢wdm｣:  
ℹ ｢wdm｣: Compiled successfully.
```